public/js/admin.js

```diff
@@ -344,7 +380,6 @@ var submitQuestionForm = function() {
             } else if (data == 'success') {
                 $('#result').html('Question added to database');
                 setTimeout(displayQuestionTable, 1000);
```

Comment:
why do we need this, and why did you remove the modal hide

Reply:
I'm not using modal anymore for creating questions, and seTimeout was there from previous revision